pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -75,6 +75,9 @@ func (p *podAffinityPriorityMap) processTerm(term *v1.PodAffinityTerm, podDefini
 	if match {
 		for _, node := range p.nodes {
 			if priorityutil.NodesHaveSameTopologyKey(node, fixedNode, term.TopologyKey) {
+				if p.counts[node.Name] == nil {
```

Comment:
It's intentional to only init the entry when needed because initilizing a `*int64` has overhead - it was captured in the perf tests.

Reply:
I am looking through current code to see if initialization of the counters misses some case(s).